kafkaesque-core/src/main/java/in/rcard/kafkaesque/config/KafkaesqueConfigLoader.java

```diff
@@ -0,0 +1,18 @@
+package in.rcard.kafkaesque.config;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigBeanFactory;
+import com.typesafe.config.ConfigFactory;
+
+public class KafkaesqueConfigLoader {
+
+    public static KafkaesqueConsumerConfig loadConsumerConfig() {
+        Config config = ConfigFactory.load();
```

Comment:
No configuration of the path of the file.

Reply:
Would the [standard behavior](https://github.com/lightbend/config#standard-behavior) be problematic ? Or is it favorable to make the path explicit :thinking:

Reply:
If there is a "standard behavior, " you must explicitly add it to the documentation. The user of `kafkaesque` should not try to understand which library we use and learn how to use that library on his own.